extension-manifest-v3/src/background/stats.js

```diff
@@ -79,6 +87,58 @@ const setIcon = throttle(
   __PLATFORM__ === 'firefox' ? 1000 : 250,
 );
 
+export async function getStats(since = '') {
+  let entries = await dailyStats.getAll();
```

Comment:
after few months, will this still be fast?

Reply:
At maximum, we will have there 1827 keys (if a user would use the browser every single day for 5 years). We are keeping in the local storage in MV2 a much bigger set of data, so this is not the case. 

Also, in MV3 we are "getting" specific keys (opposite to whole in MV2 - options are in the root object), so daily stats will be serialized for now, only if a new tab extension needs it.

Reply:
I assume you could try to fill it to capacity to simulate the worst case. And then see if it would create issues. If the operation is not a critical path (or during startup), it might not be a problem.

Reply:
it's not only about the data size but also write frequency

Reply:
The `chrome.storage.local.get()` is an async operation, so it does not block the execution of the service-worker sync operations.